drivers/pinctrl/pinctrl_gecko.c

```diff
@@ -33,6 +33,15 @@ int pinctrl_configure_pins(const pinctrl_soc_pin_t *pins, uint8_t pin_cnt, uintp
 		pin_config.pin = GECKO_GET_PIN(pins[i]);
 		loc = GECKO_GET_LOC(pins[i]);
 
+		/* Force input for low power mode */
+		if (GECKO_GET_LP(pins[i]) == GECKO_LP_ENABLE) {
+			pin_config.mode = gpioModeInput;
+			pin_config.out = 0;
+			GPIO_PinModeSet(pin_config.port, pin_config.pin, pin_config.mode,
+					pin_config.out);
+			continue;
+		}
```

Comment:
I am a bit annoyed `psels` is ignored if `low-power-enable` is set.

Reply:
I could suggest to encapsulate GPIO_PinModeSet with all flags passing and low power flags will proceed his own pin control settings with a switch case to be psels aware. Do you prefer that way?

Reply:
I have read a bit more about `GPIO_PinModeSet()`. I am now a bit confused. What do you expect when `low-power-enable` is set?